content/docs/deployment/csmoperator/drivers/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "CSI Drivers"
+linkTitle: "CSI Drivers"
+description: CSI Drivers
+weight: 1
```

Comment:
@tdawe : Installation of CSI Drivers using CSM Operator

Reply:
Fixed